src/test/java/org/mule/module/apikit/validation/QueryStringRequestValidator.java

```diff
@@ -36,9 +35,61 @@ public void validQueryString() throws MuleRestException {
   }
 
   @Test
-  public void invalidQueryString() throws MuleRestException {
-    expectedException.expect(InvalidQueryStringException.class);
-    expectedException.expectMessage("Invalid value for query string");
+  public void notValidAdditionalParamQueryString() {
+    MultiMap<String, String> queryParams = new MultiMap<>();
+    queryParams.put("start", "2");
+    queryParams.put("lat", "12");
+    queryParams.put("long", "13");
+    queryParams.put("extraParam", "13");
+    testRestRequestValidatorBuilder
+        .withApiLocation("unit/query-string/api.raml")
+        .withMethod("GET")
+        .withRequestPath("/api/locations")
+        .withRelativePath("/locations")
+        .withQueryParams(queryParams)
+        .withQueryString("start=2&lat=12&long=13")
```

Comment:
extraParam should be here despite the fact that it does not seem to have any impact